tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py{35,36,37,38,39}-{ubuntu-latest,macos-latest,windows-latest},lint
 usedevelop = true
 deps =
     docker
+    google-cloud-logging
```

Comment:
This needs added as a dependency in `setup.py` instead: https://github.com/GoogleCloudPlatform/functions-framework-python/blob/04423e7046bc73bdf82e63da176d3dd6aac88d6b/setup.py#L51-L57

Reply:
Should we force this as a dependency for every user? This is only needed for GCF Python 3.7, so I put the imports under the if statement.

Reply:
You can limit this to just Python 3.7 with a dependency specifier: https://www.python.org/dev/peps/pep-0508/#environment-markers, but it still needa to be in `setup.py` -- this file is just for the test framework.